web/packages/teleport/src/services/nodes/nodes.test.ts

```diff
@@ -31,7 +31,6 @@ test('correct formatting of nodes fetch response', async () => {
         hostname: 'im-a-nodename',
         labels: [{ name: 'env', value: 'dev' }],
         addr: '192.168.86.132:3022',
-        subKind: 'SSH Server',
```

Comment:
I know we default to `SSH Server` now, but since the API always returns a subkind we should make sure this mock API is returning it too, yeah?

Reply:
Yes, added to the API mock.